.circleci/config.yml

```diff
@@ -937,69 +903,129 @@ workflows:
           filters:
             branches:
               only: master
+
+  bors-auto-workflow:
```

Comment:
Bors is explicitly looking for a workflow titled `commit-workflow` so if we land this as is no PR will be able to land since the check bors is looking for wont ever happen. We can change this, it would just require re-deploying bors with a config change.

Reply:
No problem, changing.   I thought bors was looking at all git checks on the pr?